doc/styles/Vocab/ANSYS/accept.txt

```diff
@@ -4,6 +4,7 @@
 [Ee]igensolver
 [Hh]yperelasticity
 [Kk]eypoints
+[Mm]apdl

```

Comment:
Actually this should be only `MAPDL`, because I think in prose `mapdl` is not accepted.